test_runner/src/main/kotlin/ftl/gc/GcTestMatrix.kt

```diff
@@ -33,7 +33,7 @@ object GcTestMatrix {
     //        "message" : "The service is currently unavailable.",
     //        "status" : "UNAVAILABLE"
     //    }
-    fun refresh(testMatrixId: String, args: IArgs): TestMatrix {
+    suspend fun refresh(testMatrixId: String, args: IArgs): TestMatrix {
```

Comment:
Simplifying is right way, I like it. You have changed signature of `cancel` function but you forgot to do the same think here.